authentik/events/models.py

```diff
@@ -620,8 +620,9 @@ class SystemTask(SerializerModel, ExpiringModel):
     name = models.TextField()
     uid = models.TextField(null=True)
 
-    start_timestamp = models.FloatField()
-    finish_timestamp = models.FloatField()
+    start_timestamp = models.DateTimeField()
+    finish_timestamp = models.DateTimeField()
+    duration = models.FloatField()
```

Comment:
I really want to play with https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.GeneratedField for this, but we'd loose precision :/

Reply:
TIL about that field, I wonder if we could use that for `User.uid`